utility/astring.cpp

```diff
@@ -8,10 +8,10 @@
 #include "fcintl.h"
 
 // Qt
-#include <QString>
-#include <QStringList>
-#include <QVector>
-#include <Qt>
+#include <QStringLiteral>
+#include <QVector>         // IWYU pragma: keep
+#include <Qt>              //Qt::SkipEmptyParts
+#include <qcontainerfwd.h> // QVector<QString>
```

Comment:
Do you really need this one, since you include `QVector` in full above?

Reply:
I get some weird circular errors. If I only include Vector and not the Container I get tidy errors that Vector isn't directly included. If I don't include Vector and include the Container, then the I get compiler errors. I added the pragma keep with both included to suppress the not directly used warning.